MNE/disp3D/helpers/types.h

```diff
@@ -112,7 +112,8 @@ namespace BrainTreeMetaItemTypes
                     RTDataNormalizationValue = QStandardItem::UserType + 115,
```

Comment:
it is possible that you only assign the first enum type QStandardItem::UserType + 116; following types don't need an explicit type they are incremented automatically